[2025-05-16T10:03:22.132+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-16T10:03:22.140+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DYNAMIC_FILE_TO_DB_LOADER.get_required_files_and_download scheduled__2025-05-16T09:40:00+00:00 [queued]>
[2025-05-16T10:03:22.143+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DYNAMIC_FILE_TO_DB_LOADER.get_required_files_and_download scheduled__2025-05-16T09:40:00+00:00 [queued]>
[2025-05-16T10:03:22.143+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-05-16T10:03:22.148+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_required_files_and_download> on 2025-05-16 09:40:00+00:00
[2025-05-16T10:03:22.151+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=11891) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-05-16T10:03:22.151+0000] {standard_task_runner.py:72} INFO - Started process 11892 to run task
[2025-05-16T10:03:22.151+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'DYNAMIC_FILE_TO_DB_LOADER', 'get_required_files_and_download', 'scheduled__2025-05-16T09:40:00+00:00', '--job-id', '1566', '--raw', '--subdir', 'DAGS_FOLDER/***_dynamic_file_to_db_loader.py', '--cfg-path', '/tmp/tmp2xnbf915']
[2025-05-16T10:03:22.152+0000] {standard_task_runner.py:105} INFO - Job 1566: Subtask get_required_files_and_download
[2025-05-16T10:03:22.172+0000] {task_command.py:467} INFO - Running <TaskInstance: DYNAMIC_FILE_TO_DB_LOADER.get_required_files_and_download scheduled__2025-05-16T09:40:00+00:00 [running]> on host 65a84f9e3a41
[2025-05-16T10:03:22.203+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DYNAMIC_FILE_TO_DB_LOADER' AIRFLOW_CTX_TASK_ID='get_required_files_and_download' AIRFLOW_CTX_EXECUTION_DATE='2025-05-16T09:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-16T09:40:00+00:00'
[2025-05-16T10:03:22.204+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-16T10:03:22.212+0000] {base.py:84} INFO - Retrieving connection 'minioic'
[2025-05-16T10:03:22.213+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='minioic', conn_type='aws') credentials retrieved from login and password.
[2025-05-16T10:03:22.315+0000] {s3.py:1467} INFO - Downloading source S3 file from Bucket ahajiyev with path abc.xlsx
[2025-05-16T10:03:22.352+0000] {httpchecksum.py:481} INFO - Skipping checksum validation. Response did not contain one of the following algorithms: ['crc32', 'sha1', 'sha256'].
[2025-05-16T10:03:22.355+0000] {s3.py:1467} INFO - Downloading source S3 file from Bucket ahajiyev with path xyz.xlsx
[2025-05-16T10:03:22.364+0000] {httpchecksum.py:481} INFO - Skipping checksum validation. Response did not contain one of the following algorithms: ['crc32', 'sha1', 'sha256'].
[2025-05-16T10:03:22.365+0000] {hook.py:208} WARNING - Data lineage tracking is disabled. Register a hook lineage reader to start tracking hook lineage.
[2025-05-16T10:03:22.366+0000] {airflow_dynamic_file_to_db_loader.py:162} INFO - Downloaded Mappings: [{'source_file_name': 'abc.xlsx', 'target_table_name': 'abc'}, {'source_file_name': 'xyz.xlsx', 'target_table_name': 'xyz'}]
[2025-05-16T10:03:22.366+0000] {python.py:240} INFO - Done. Returned value was: [{'source_file_name': 'abc.xlsx', 'target_table_name': 'abc'}, {'source_file_name': 'xyz.xlsx', 'target_table_name': 'xyz'}]
[2025-05-16T10:03:22.379+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-16T10:03:22.380+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=DYNAMIC_FILE_TO_DB_LOADER, task_id=get_required_files_and_download, run_id=scheduled__2025-05-16T09:40:00+00:00, execution_date=20250516T094000, start_date=20250516T100322, end_date=20250516T100322
[2025-05-16T10:03:22.425+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-16T10:03:22.449+0000] {taskinstance.py:3901} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-05-16T10:03:22.450+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
